Gordon360/Services/NewsService.cs

```diff
@@ -169,7 +169,7 @@ public StudentNews SubmitNews(StudentNews newsItem, string username)
                 var serverAddress = _serverUtils.GetAddress();
                 if (serverAddress is not string) throw new Exception("Could not upload Student News Image: Server Address is null");
 
-                var url = $"{serverAddress}/browseable/uploads/{filename}";
+                var url = $"{serverAddress}/browseable/uploads/news/{filename}";
```

Comment:
It might be worth pulling this out into a separate (private) function as well, so that all the references to the server path are consistent